azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/legacy/function/handlers/AnnotationHandlerImpl.java

```diff
@@ -123,8 +128,21 @@ private Retry getRetryConfigurationFromMethod(Method method) throws AzureExecuti
     }
 
     protected void processParameterAnnotations(final Method method, final List<Binding> bindings) {
+        SdkParameterAnalyzer analyzer = new SdkParameterAnalyzer();
+        SdkParameterAnalysisResult result = analyzer.analyze(method);
+        Set<Parameter> sdkTypeParams = new HashSet<>();
+        for (final SdkTypeMetaData sdkTypeMetaData : result.getSdkTypesMetaData()) {
+            sdkTypeParams.add(sdkTypeMetaData.getParam());
+        }
+
         for (final Parameter param : method.getParameters()) {
             bindings.addAll(parseAnnotations(param::getAnnotations, this::parseParameterAnnotation));
+
+            if (sdkTypeParams.contains(param)) {
```

Comment:
Should we add `supportsDeferredBinding` to all bindings or binding related with param?

Reply:
Got it, if so, I think we should move below logic to `parseAnnotations`, if we put it here it may also add 
`supportsDeferredBinding` to previous bindings
```
                Map<String, String> props = new HashMap<>();
                props.put("supportsDeferredBinding", "true");
                bindings.forEach(binding -> binding.setAttribute("properties", props));
```